tests/phpunit/tests/functions/wpRemovableQueryArgs.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Tests for the wp_parse_list() function.
+ *
+ * @group functions
+ *
+ * @covers ::wp_removable_query_args
+ */
+class Tests_Functions_wpRemovableQueryArgs extends WP_UnitTestCase {
+
+	/**
+	 * Test that the wp_removable_query_args() function returns an array.
+	 *
+	 * This test verifies that the function wp_removable_query_args() returns a valid array.
+	 *
+	 * @ticket 59938
+	 */
+	public function test_should_return_array_when_called() {
+		$result = wp_removable_query_args();
+		$this->assertIsArray( $result, 'The return value is not an array.' );
+	}
+
+	/**
+	 * Test that the array returned by wp_removable_query_args() is not empty.
+	 *
+	 * This test ensures that wp_removable_query_args() returns an array that contains
+	 * query arguments and is not an empty array.
+	 *
+	 * @ticket 59938
+	 */
+	public function test_should_return_non_empty_array_when_called() {
+		$result = wp_removable_query_args();
+		$this->assertNotEmpty( $result, 'The returned array is empty.' );
+	}
+
+	/**
+	 * Test that the filter applied to wp_removable_query_args() works as expected.
+	 *
+	 * This test ensures that the custom filter function applied via `add_filter()`
+	 * correctly modifies the array returned by wp_removable_query_args() by adding
+	 * a custom query argument.
+	 *
+	 * @ticket 59938
+	 */
+	public function test_should_modify_array_when_filter_applied() {
+		// Add a custom query argument using a filter
+		add_filter(
+			'removable_query_args',
+			function ( $args ) {
+				$args[] = 'custom_arg';
+				return $args;
+			}
+		);
+
+		$result = wp_removable_query_args();
+
+		// Assert that the custom argument is in the array
+		$this->assertContains( 'custom_arg', $result, 'The filter did not modify the array as expected.' );
+
+		// Remove the filter after the test
```

Comment:
```suggestion
		// Remove the filter after the test.
```

WPCS fix.

Or omit this comment, it is just stating the obvious.